tests/io/feff/test_inputs.py

```diff
@@ -390,3 +390,9 @@ def test_paths_string(self):
         ]
         answer = "\n".join(lines)
         assert answer == str(self.paths)
+
+
+def test_atom_num(self):
+    atoms = Atoms.cluster_from_file("ATOMS_test.inp")
+    num_atom = 37
+    assert len(atoms) != num_atom
```

Comment:
why not test for the expected number of atoms?

Reply:
Hi Janosh,
The expected number of atoms is 37; how should I specify this value?

Reply:
Is `!=` a typo?

Reply:
You're missing one level of indentation to make it a method of the test class.

Reply:
Let me fix it. sorry to confuse you. I want it to pass when the expected number of atoms is equal to num_atom and throw an error when the numbers are not equal.